### DISCUSSION TITLE
Suggestion: Show QSO listing when clicking on cell in DXXC tab in QSO Detail window

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#167] [Ideas]
Hi Ladislaw, 
This request quite similar to my request on the Award Window (#166)

Clicking on the cells in DXCC tab in QSO detail to show previous qsos with the entity on band/mode in the Log book view window. 

73 de LB2EG Richard

![Screenshot from 2023-01-22 14-53-04](https://user-images.githubusercontent.com/22942440/213919569-28625d8a-80b9-4bc9-8226-2834d3ab2a51.png)

**Comment by foldynl** [comment]
Hi Richard,
it is important if this will be of any use. Can you please tell me under what circumstances I should use this new feature? Your notes from #166 makes sense to me, because you want to see the detail of the DXCC country. But if I'm looking at the details of a QSO, why don't I want to see all the QSOs for given country? It doesn't make much sense to me to implement this thing in QSO Detail, but maybe you can tell me some use-case in which it can be used.

**Reply by rsaue** [reply]
Hi, 
the details of the whole dxcc entity is not that important. 

The detail info on the whole DXCC entity would be convenient to look at with one click if i have rather few coloured cells and I want to get an overview of all QSOs with the entity in one view, e.g. to see if a station is my own log has responded to QSL / LOTW reguests on earlier QSOs, or maybe that track record is better with other stations in log. 

What is more important is the details related to the cells. Having this opportunity when looking on a QSO the QSO detail window, would be a nice feature when the operator is using his time looking at the DXCC Tab, realizing there is not confirmed QSOs on a band or a mode. It would be a not so convenient (many clicks) closing the QSO detail window, opening the Awards view window, look up the entity and click on the cell there to get this information. 

73 de LB2EG Richard

**Reply by foldynl** [reply]
Hello,

I'm just implementing and testing your idea #168  and it makes sense to me. It is beautiful, user-intuitive and very useful. 

Unfortunately, I am not sure about this idea (#167) because it brings together two things that are not relate to each other. On the one side, there is a QSO detail, on the other side, there should be DXCC Entity/band filter. These are two different actions that are unrelated and in my opinion should not be combined.

Please, could you try to describe to me more about this idea. Instead of writing that it will save one or two clicks or it is already implemented in another software, I would rather like to know the thoughts that lead to this proposal.  

Perhaps it is needed to improve QSL management and its display in QSO Detail. Maybe it would be nice to show clearly in the QSO Detail that it is a Confirmed/Unconfirmed QSO instead of showing all QSOs from particular DXCC Entity when you click on QSO Detail DXCC tab . What do you about it?

It is also necessary to think about operators who work on "small" displays, that they do not have to see filtered QSOs, because the logbook will be covered by the QSO Detail dialog.
